src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
+///
+/// - be consistently thinner and a bit further away from the link text
+/// - have a clearer hover state, where the underline gets thicker to make the
+///   link stand out to users
+///
+/// These changes are an opt-in feature because you may:
+///
+/// - need to make changes to your service to introduce these new link styles
+///   consistently
+/// - want to avoid opting in if you have links within a multiple-column layout
+///   in your CSS - there is [a Chromium bug that affects such
+///   links](https://github.com/alphagov/govuk-frontend/issues/2204)
+///
+/// These new link styles will be enabled by default in a future major version
+/// of GOV.UK Frontend.
+///
 /// @type Boolean
 /// @access public
 
 $govuk-new-link-styles: false !default;
 
-/// Link underline thickness
+/// Thickness of link underlines
+///
+/// The default is whichever is calculated to be thicker:
 ///
-/// The default is, for a given link, whichever is thicker:
-///  - an absolute 1px underline
-///  - .0625rem, equivalent to 1px but adjusts with the root font size if the
-///.   user has zoomed in 'text only'
+///  - 1px
+///  - 0.0625rem (equivalent to 1px at 16px root font size, but scales if the
+///    user zooms in 'text only')
 ///
-/// Set to false to disable setting a thickness
+/// Set this variable to `false` to avoid setting a thickness.
 ///
 /// @type Number
 /// @access public
 
 $govuk-link-underline-thickness: unquote("max(1px, .0625rem)") !default;
 
-/// Link underline offset.
+/// Offset of link underlines from text baseline
 ///
-/// Set to false to disable setting an offset
+/// Set this variable to `false` to avoid setting an offset.
 ///
 /// @type Number
 /// @access public
 
 $govuk-link-underline-offset: .1em !default;
 
-/// Link hover state underline thickness
+/// Thickness of link underlines in hover state
+///
+/// The default is, for a given link, whichever is calculated to be thicker:
```

Comment:
Suggest 'The default for each link is' rather than 'The default is, for a given link,".

This also needs a tweak to make the bullet lead-in work, as above.

Maybe: "The default for each link will be the thickest of the following:"